version/1/4/fr/code_of_conduct.md

```diff
@@ -25,18 +25,18 @@ Exemples de comportements non acceptables :
 * l'utilisation de langage ou d'imagerie sexualisés et les avances sexuelles
 non sollicitées
 * le _trolling_, les commentaires insultants ou désobligeants, et les
-attaques personnelles ou d'ordre politique
+attaques personnelles ou politiques
 * le harcèlement en public ou en privé
 * la publication d'informations privées de tierces personnes, telles que
-  des adresses physiques ou électroniques, sans permission explicite
+  les adresses physiques ou électroniques, sans permission explicite
```

Comment:
"any addresses" -> "the addresses"
I think the change is minor, but I'd keep the older version.